op-scim-bridge/schema.yaml

```diff
@@ -2,7 +2,7 @@ x-google-marketplace:
   schemaVersion: v2
   applicationApiVersion: v1beta1
   # publishedVersion should match the value of appVersion in the Chart.yaml
```

Comment:
```suggestion
  # publishedVersion should match the value of appVersion in the Chart.yaml and should equal the docker image version
```

Reply:
addressed in 91d0beebf3b3222cce953cc7e8a11a01ae96ae33